Yoast/yoast-components
# Added id to draft js editor field

## Summary

This PR can be summarized in the following changelog entry:

* Adds an id to the draft JS editor `editablecontent` element.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Make sure this branch is checked out and `yoast-components` is linked to `wordpress-seo`.
* On the post edit page open the snippet editor.
* Inspect the source and look for the div with id `snippet-editor-field-title`. This was the existing id on the container wrapper.
* Now look for the div with id `snippet-editor-field-title-field`. This is the newly added id. Make sure the same div has the `contenteditable` attribute.
* The same can be done for `snippet-editor-field-description` and `snippet-editor-field-title-field`.

Fixes #637
